official/utils/arg_parsers/parsers.py

```diff
@@ -142,16 +142,29 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
     if batch_size:
       self.add_argument(
           "--batch_size", "-bs", type=int, default=32,
-          help="[default: %(default)s] Batch size for training and evaluation.",
+          help="[default: %(default)s] Global batch size for training and "
+               "evaluation.",
           metavar="<BS>"
       )
 
+    assert not (multi_gpu and num_gpu)
+
     if multi_gpu:
       self.add_argument(
           "--multi_gpu", action="store_true",
           help="If set, run across all available GPUs."
       )
 
+    if num_gpu:
+      self.add_argument(
+          "--num_gpus", "-ng",
+          type=int,
+          default=1 if tf.test.is_built_with_cuda() else 0,
+          help="[default: %(default)s] How many GPUs to use with the "
+               "DistributionStrategies API.",
```

Comment:
Can you add a note here that reflects the details in the readme? Specifically, that 0==CPU, 1==GPU, default is what you built TF with.